README.md

```diff
@@ -20,19 +20,19 @@ Choose an overview of your container infrastructure, or focus on a specific micr
 
 <img src="imgs/selected.png" width="200" alt="Focus on a single container" align="right">
 
-View contextual metrics, tags and metadata for your containers.  Effortlessly navigate between processes inside your container to hosts your containers run on, arranged in expandable, sortable tables.  Easily find the container using the most CPU or memory for a given host or service.
+View contextual metrics, tags, and metadata for your containers.  Effortlessly navigate between processes inside your container to hosts your containers run on, arranged in expandable, sortable tables.  Easily find the container using the most CPU or memory for a given host or service.
 
 ### Interact with and manage containers
 
 <img src="imgs/terminals.png" width="200" alt="Launch a command line." align="right">
 
-Interact with your containers directly: pause, restart and stop containers. Launch a command line. All without leaving the scope browser window.
+Interact with your containers directly: pause, restart, and stop containers. Launch a command line. All without leaving the scope browser window.
 
 ### Extend and customize via plugins
 
-Add custom details or interactions for your hosts, containers and/or processes by creating Scope plugins; or just choose from some that others have already written at the Github [Weaveworks Scope Plugins](https://github.com/weaveworks-plugins/) organization.
+Add custom details or interactions for your hosts, containers, and/or processes by creating Scope plugins. Or, just choose from some that others have already written at the GitHub [Weaveworks Scope Plugins](https://github.com/weaveworks-plugins/) organization.
 
-## <a name="getting-started"></a>Getting started
```

Comment:
Not a strong opinion, but I would leave this "title case".

Reply:
"Getting Started" would be title case (in English).

Reply:
Entirely right, yes!